packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_row.dart

```diff
@@ -85,22 +78,25 @@ class _LoggingTableRowState extends State<LoggingTableRow> {
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
                 RichText(
-                  text: LoggingTableRow._detailsSpan(
-                    context,
-                    widget.data.asLogDetails(),
+                  text: TextSpan(
+                    text: widget.data.asLogDetails(),
+                    style: Theme.of(context).regularTextStyle,
                   ),
                 ),
                 Row(
                   children: [
                     RichText(
-                      text: LoggingTableRow._metadataSpan(
-                        context,
-                        'Some METADATA',
+                      text: TextSpan(
+                        text: 'Some META data',
+                        style: Theme.of(context).subtleTextStyle,
```

Comment:
use the meta data style from above. This will ensure consistency with what we are using and what we are measuring